stable/helm-exporter/templates/deployment.yaml

```diff
@@ -2,11 +2,16 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "helm-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Helm supplies this does it not?  Why do we need to specify this value?

Reply:
Helm will supply on an install. This allows helm package to be used. Followed the convention from https://github.com/helm/charts/blob/master/stable/grafana/templates/deployment.yaml

Reply:
Not sure what you mean by `allows helm package to be used`